index.html

```diff
@@ -5,330 +5,346 @@
+                    <div class="skills">
+                        <a href="#" class="skillscontainer">
+                            <h4>Skills</h4> <img class="arrow2" src="./images/right-arrow.png" alt="">
+                        </a>
+                    </div>
+                    <div class="line">
+                        <img class="subs" src="./images/subtract.png" alt="">
+                    </div>
                 </div>
             </div>
-            <div class="container6">
-                <div class="languages">
-                    <a href="#" class="langcontainer">
-                        <h4>Languages</h4> <img class="arrow3" src="./Images/right-arrow.png" alt="">
-                    </a>
+            <div class="cont7">
+                <div class="contact">
+                    <h2>Contact Me</h2>
                 </div>
-
-                <ul class="last">
-                    <li class="js">
-                        <a href="#"> <img class="EJS" src="./Images/Frame9.png" alt="#">
-                        </a>
-                    </li>
-                    <li class="html">
-                        <a href="#"> <img class="EH" src="./Images/Frame10.png" alt="#">
-                        </a>
-                    </li>
-                    <li class="css">
-                        <a href="#"> <img class="CS" src="./Images/Frame11.png" alt="#">
-                        </a>
-                    </li>
-                </ul>
-                <div class="frameworks">
-                    <a href="#" class="framecontainer">
-                        <h4>Frameworks</h4> <img class="arrow" src="./Images/right-arrow.png" alt="">
-                    </a>
+                <div class="context">
+                    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br> love to help with it </p>
                 </div>
-                <div class="line">
-                    <img class="subs" src="./Images/subtract.png" alt="">
+                <div class="forms">
+                    <form class="form" action="index.html" method="post">
+                        <label for="name"></label>
+                        <input type="text" id="name" name="Name" placeholder="Enter Name" required maxlength="30"><br><br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="jdoe@email.com" placeholder="jdoe@email.com" required><br><br>
+                        <div class="areas">
+                            <textarea name="textarea" id="texta" cols="30" rows="10" placeholder="Write your message here" required maxlength="500"></textarea>
+                        </div>
+                        <div class="touch">
+                            <button class="resumetext" type="submit" name="button">Get In Touch</button>
+                        </div>
+                    </form>
```

Comment:
- [ ] Kindly make sure that your form uses the action and method attribute to integrate with the Formspree service. Your `action` value should be similar to `https://formspree.io/f/{form_id}`.
- [ ] Kindly stick to the design template in Figma as much as possible, this includes the font, colors, and background colors(your form has some black borders around it). Also, please make sure to add the required interaction to the form submit button.
- Your form UI
![image](https://user-images.githubusercontent.com/33767951/139856237-32cf061c-141b-495c-946f-190b537044bc.png)
- Required form UI
![image](https://user-images.githubusercontent.com/33767951/139856840-a79b6bb6-5fe6-4b6f-b49e-58679639b805.png)